### PR TITLE
README: update tidb package build.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Read the [Roadmap](./ROADMAP.md).
 
 Go environment. Currently a 64-bit version of go >= 1.5 is required.
 ```
-go get -d github.com/pingcap/tidb
+git clone https://github.com/pingcap/tidb.git $GOPATH/src/github.com/pingcap/tidb
```
Comment (siddontang) [inline README.md]: use `cd $GOPATH` before?

Comment (qiuyesuifeng) [inline README.md]: needless, for we can use git to clone to `$GOPATH/src/github.com/pingcap/tidb`.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**